### DISCUSSION TITLE
st->stikp[i].localCOPPosのY成分

### DISCUSSION THREAD

**Discussion by Naoki-Hiraoka** [YutaKojio/hrpsys-base#32] [Q&A]
st->stikp[i].localCOPPosのY成分に足す値が0なのには何か事情があったのでしょうか
https://github.com/YutaKojio/hrpsys-base/blob/5aab7f9809b1ef76dd62bf323e522834d84097be/rtc/AutoBalancer/AutoBalancer.cpp#L1192

**Comment by YutaKojio** [comment]
https://github.com/fkanehiro/hrpsys-base/pull/661/commits/b32d4c68b5eb197662db098875f8e6da19f9c384
僕も不思議だったんだけど、最初はmasterとのdiffを減らしたくて変えてないまま今も残ってました。
PRの日付見るとDRC直前だし大会用のhotfixで正しくなかった可能性もある。

**Comment by YutaKojio** [comment]
localCOPPosが影響するのがZMP分配のところだと思うけど，左右対称なら0にしてもあまり大きな変化はないような気がする．
わざわざ0にしてるからなにかしたら意味があったんだと思うけど，正しさでいうとyもセットするのが正しいと思うから，実機で確認してみて問題なさそうならyもセットするように変更するのでいいかな？
